README.md

```diff
@@ -0,0 +1,42 @@
+# Pachyderm Helm Chart
+
+The repo contains the pachyderm helm chart.
+
+Status: **Experimental**
+
+## Folder Structure
+
+```
+pachyderm - the helm chart itself
+test - Go based tests for the helm chart
+```
+
+## Developer Guide
+To run the tests for the helm chart, run the following:
+
+```
+make test
+```
+
+# Diff against `pachctl`
+
+The scheme here is to take a recent `pachctl` and diff its manifest &
+the Helm-generated manifest.
+
+ 1. Generate a `pachctl` manifest with `pachctl deploy google blah 10
+    --dynamic-etcd-nodes 1 -o yaml --dry-run > pachmanifest.yaml`
+ 2. Generate a Helm manifest with `helm template -f
+    examples/gcp-values.yaml ./pachyderm --generate-name >
+    helmmanifest.yaml`
+ 3. Visually diff the two.  I did this with the currently checked-in
```

Comment:
Sorry, couple small things here
- `pachctlmanifests.yaml` isn't checked in (nor should it be I think) Can you remove the reference in the readme
- probably should use a consistent name instead of `--generate-name` (not a huge deal in our case, as I don't think we really use that in the manifest much)